plugins/ipam/host-local/allocator.go

```diff
@@ -163,3 +163,37 @@ func networkRange(ipnet *net.IPNet) (net.IP, net.IP, error) {
 	}
 	return ipnet.IP, end, nil
 }
+
+// nextIP returns the next ip of curIP within ipallocator's subnet
+func (a *IPAllocator) nextIP(curIP net.IP) net.IP {
+	if curIP.Equal(a.end) {
+		return a.start
+	}
+	return ip.NextIP(curIP)
+}
+
+// getSearchRange returns the start and end ip based on the last reserved ip
+func (a *IPAllocator) getSearchRange() (net.IP, net.IP) {
+	var startIP net.IP
+	var endIP net.IP
+	startFromLastReservedIP := false
+	lastReservedIP, err := a.store.LastReservedIP()
+	if err == nil && lastReservedIP != nil {
```

Comment:
My intention is to ignore error from store.LastReservedIP, since it is not critical enough to block the whole flow. 

I should still log here. Sounds good?